src/app/utils.py

```diff
@@ -28,7 +28,7 @@
 from spdx_license_matcher.difference import get_similarity_percent
 from spdx_license_matcher.utils import get_spdx_license_text
 
-from app.models import User, UserID
+from app.models import User, UserID, LicenseRequest, LicenseNamespace
```

Comment:
Instead of LicenseRequest, it should be License on line 437 in utils and everywhere else in utils. `shortIdentifier` is a field of License not LicenseRequest